temperature/crud.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+from fastapi import HTTPException, Depends, status
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.exc import IntegrityError
+from websockets.legacy import async_timeout
+
+from temperature import utils
+from weather import models
+from weather.crud import get_all_cities
+from dependencies import get_db
+
+
+async def get_temperature_list(db: AsyncSession):
+    result = await db.execute(
+        select(models.Temperature)
+    )
+    return result.scalars()
+
+
+async def get_temperatures_by_city_id(db: AsyncSession, city_id: int):
+    result = await db.execute(
+        select(models.Temperature).filter(models.Temperature.city_id == city_id)
+    )
+    temperature = result.scalars().first()
```

Comment:
In `get_temperatures_by_city_id`, the `first()` method is used. Ensure that this behavior is intended, as it will only return the first matching record. If multiple records are expected, consider using `all()` instead.